### ISSUE TITLE
Commit 114f610 causes the Crypt plugin to error when using the filevault_escrow module

### ISSUE THREAD

**Issue by awidner** [munkireport/filevault_escrow#1]
Commit 114f610 causes the Crypt plugin to error when using the `filevault_escrow` module. The `checkin` function needs to be public in order to successfully return a 200 to Crypt as defined here: https://github.com/munkireport/filevault_escrow/blob/master/filevault_escrow_controller.php#L31-L35

Observed the following errors in `/var/log/crypt.log` on macOS 10.14.6, Crypt 3.3.0.178:

```
2019-09-18 03:58:54 PM - INFO: Stored recovery key is valid.
2019-09-18 03:58:54 PM - INFO: KeyEscrowInterval set to: 1 hour(s)...
2019-09-18 03:58:54 PM - INFO: Attempting to Escrow Key...
2019-09-18 03:58:54 PM - DEBUG: ServerURL Pref set to: https://www.company.com/index.php?/module/filevault_escrow/...
2019-09-18 03:58:54 PM - ERROR: Key escrow unsuccessful.
```

Workaround fix added to `app/controllers/module.php`:

```       
// These methods don't require authentication
$unProtectedActions = ["get_script", "index", "checkin"];
// Require authentication for all methods
if( ! in_array($this->action, $unProtectedActions) && ! $this->module_obj->authorized())
{
  $this->requestForbidden('Module controller filter');
 }
```

When `checkin` was added to `$unProtectedActions` in `app/controllers/module.php` the Crypt plugin started logging normally:

```
2019-09-18 04:04:57 PM - INFO: Stored recovery key is valid.
2019-09-18 04:04:57 PM - INFO: KeyEscrowInterval set to: 1 hour(s)...
2019-09-18 04:04:57 PM - INFO: Attempting to Escrow Key...
2019-09-18 04:04:57 PM - DEBUG: ServerURL Pref set to: https://www.company.com/index.php?/module/filevault_escrow/...
2019-09-18 04:04:57 PM - INFO: Key escrow successful.
```

EDIT: update code blocks for readability

**Comment by bochoven** [comment]
I think you also should be able to change the url to 

```
sudo defaults write /Library/Preferences/com.grahamgilbert.crypt ServerURL "http://munkireportURL/index.php?/module/filevault_escrow/index/"
```

**Comment by awidner** [comment]
Looks like changing the URL works:

```
19-09-19 12:51:32 PM - INFO: Stored recovery key is valid.                                                                                                                                
2019-09-19 12:51:32 PM - INFO: KeyEscrowInterval set to: 1 hour(s)...                                                                                                                       
2019-09-19 12:51:32 PM - INFO: Attempting to Escrow Key...                                                                                                                                  
2019-09-19 12:51:32 PM - DEBUG: ServerURL Pref set to: https://www.company.com/index.php?/module/filevault_escrow/...                                                         
2019-09-19 12:51:33 PM - ERROR: Key escrow unsuccessful.                                                                                                                                    
2019-09-19 12:53:33 PM - INFO: Stored recovery key is valid.                                                                                                                                
2019-09-19 12:53:33 PM - INFO: KeyEscrowInterval set to: 1 hour(s)...                                                                                                                       
2019-09-19 12:53:33 PM - INFO: Attempting to Escrow Key...                                                                                                                                  
2019-09-19 12:53:33 PM - DEBUG: ServerURL Pref set to: https://www.company.com/index.php?/module/filevault_escrow/index/...                                                   
2019-09-19 12:53:33 PM - INFO: Key escrow successful.                                                                                                                                       
2019-09-19 12:53:33 PM - INFO: Ensuring permissions on plist.                                                                                                                               
2019-09-19 12:55:34 PM - INFO: Stored recovery key is valid.                                                                                                                                
2019-09-19 12:55:34 PM - INFO: KeyEscrowInterval set to: 1 hour(s)...                                                                                                                       
2019-09-19 12:55:34 PM - INFO: We escrowed less than 1 hour(s) ago. Skipping... 
```

**Comment by awidner** [comment]
@bochoven Would you now recommend submitting a PR to update the README?

**Comment by bochoven** [comment]
Addressed here: e91c26f81751e93de7890b94f079af287c180b60 Thanks for reporting!
